src/python/pants/core/util_rules/system_binaries.py

```diff
@@ -35,7 +36,7 @@
 # -------------------------------------------------------------------------------------------
 
 # TODO(#14492): This should be configurable via `[system-binaries]` subsystem, likely per-binary.
-SEARCH_PATHS = ("/usr/bin", "/bin", "/usr/local/bin")
+SEARCH_PATHS = ("/usr/bin", "/bin", "/usr/local/bin", "/opt/homebrew/bin/")
```

Comment:
This is an actual user change, not only test fix. Maybe worth a dedicated PR so it shows up in the changelog? This could have actual impact on users, that we now choose a Brew binary rather than system binary

Reply:
We look for binaries in search path order, so surely this would only mean that we would find a brew binary if there was no system binary?

Reply:
https://github.com/pantsbuild/pants/pull/17411